content/docs/components/serving/seldon.md

```diff
@@ -47,6 +47,115 @@ spec:
 
 Save the above resource and apply it with `kubectl`.
 
-## Examples
+## Simple Example
 
-Seldon provides a [large set of example notebooks](https://docs.seldon.io/projects/seldon-core/en/latest/examples/notebooks.html) showing how to run inference code for a wide range of machine learning toolkits.
+
+Create a new namespace:
+
+```
+kubectl create ns testseldon
+```
+
+Label that namespace so we can run inference tasks in it:
```

Comment:
Change "we" to "you". The word "we" is confusing, as the reader doesn't know who it refers to.